src/edu/washington/escience/myria/api/encoding/PerfEnforceStatisticsEncoding.java

```diff
@@ -0,0 +1,40 @@
+/**
+ *
+ */
+package edu.washington.escience.myria.api.encoding;
+
+import java.util.List;
+
+/**
+ * This class encodes statistical information about a table from the user's schema.
+ */
+public class PerfEnforceStatisticsEncoding {
+  @Required public String table_name;
+  @Required public long table_size;
+  @Required public String selectivity_predicate_001;
+  @Required public String selectivity_predicate_01;
+  @Required public String selectivity_predicate_1;
+
+  public PerfEnforceStatisticsEncoding(
+      final String table_name, final long table_size, final List<String> selectivityList) {
```

Comment:
I think we can just merge these two constructors since the second one is never called except by the first one.